scripts/scil_visualize_connectivity.py

```diff
@@ -180,11 +185,12 @@ def main():
         if len(x_ticks) != len(x_legend) \
                 or len(y_ticks) != len(y_legend):
             logging.warning('Legend is not the same size as the data.'
-                            'Make sure you are using the same reordering json.')
-        plt.xticks(x_ticks, x_legend,
+                            'Make sure you are using the same reordering '
+                            'json.')
+        plt.xticks(y_ticks, y_legend,
```

Comment:
Can you fix the comments that I added in the original Max's PR ?

Reply:
done